README.md

```diff
@@ -15,3 +16,47 @@
 
 ### Run unit tests
     yarn test:unit
+
+## Using this library
+
+Package requirements, to be added after `vue create ...`
+
+```bash
+# Vuetify
+vue add vuetify
+
+# FontAwesome
+yarn add @fortawesome/fontawesome-free -D
+# OR add the CDN link in index.html
+# <link href="https://use.fontawesome.com/releases/v5.0.13/css/all.css" rel="stylesheet">
+```
```

Comment:
I think we can do away with this part; in #19 it just shows `yarn add @girder/components`, which will install vuetify, and the icon library could be invisible to downstreams depending on #20.